packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -456,6 +456,12 @@ class RenderEditable extends RenderBox {
     return _textPainter.maxIntrinsicWidth;
   }
 
+  @override
+  double computeDistanceToActualBaseline(TextBaseline baseline) {
```

Comment:
for consistency, we should put this method below the intrinsic height methods (just above hitTestSelf)